_maps/RandomRuins/SpaceRuins/spacearcade.dmm

```diff
@@ -0,0 +1,658 @@
+//MAP CONVERTED BY dmm2tgm.py THIS HEADER COMMENT PREVENTS RECONVERSION, DO NOT REMOVE
+"a" = (
+/turf/open/space/basic,
+/area/space)
+"b" = (
+/turf/closed/wall/mineral/gold,
+/area/ruin/space/has_grav/powered)
+"c" = (
+/turf/open/floor/eighties,
+/area/ruin/space/has_grav/powered)
+"d" = (
+/turf/open/floor/carpet/purple,
+/area/ruin/space/has_grav/powered)
+"e" = (
+/turf/open/floor/light/colour_cycle/dancefloor_a,
+/area/ruin/space/has_grav/powered)
+"f" = (
+/turf/open/floor/light/colour_cycle/dancefloor_b,
+/area/ruin/space/has_grav/powered)
+"g" = (
+/obj/machinery/computer{
+	name = "Dance Dance Revolution";
+	desc = "Unfortunately, the machine seems to be broken..."
+	},
```

Comment:
https://en.wikipedia.org/wiki/Dance_Dance_Revolution

Reply:
Right

Reply:
You probably shouldn't use that name directly. Change the name slightly.